Makefile

```diff
@@ -51,19 +51,19 @@ use-oncluster-controller:
 
 # Install CRDs into a cluster
 install: manifests
-	kubectl apply -f config/crds
+	kubectl apply -f config
 
 # Deploy controller in the configured Kubernetes cluster in ~/.kube/config
 deploy: manifests
-	kubectl apply -f config/crds
+	kubectl apply -f crds
 	kustomize build config/default | kubectl apply -f -
 
 # Provide CRDs that work back to k8s 1.11
 CRD_OPTIONS ?= "crd:trivialVersions=true"
 
 # Generate manifests e.g. CRD, Webhooks
 manifests:
-	${CONTROLLER_GEN} ${CRD_OPTIONS} crd rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crds/bases output:crd:dir=config/crds
+	${CONTROLLER_GEN} ${CRD_OPTIONS} crd rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crds/bases 
```

Comment:
@shawn-hurley to preserve the previous directory structure, can we add some commands here?


```
manifests:                                                                         
  ${CONTROLLER_GEN} ${CRD_OPTIONS} crd rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crds/bases 
  mv config/migration.openshift.io*yaml config/crds                                
  rm -rf config/crds/bases
```

Reply:
From what @shawn-hurley told me earlier, the yaml files under config/crd/bases may or may not always be the same as the files under config, although it's not clear that, as an intermediate output step we need to keep them once the final crds are created under config.

Reply:
My code snippet above is based on the assumption that we do not need the files in `bases` after the final output is produced.